redhat-cop/containers-quickstarts
# Add new Docker build templates to allow overriding base images

#### What is this PR About?
This PR adds a new Docker buildconfig template which allows for overriding the `FROM` clause of the target Dockerfile build. This will support resolving #186 in [Labs CI/CD](https://github.com/rht-labs/labs-ci-cd)

#### How do we test this?
Apply this template with openshift-applier to provision SonarQube and provide the DockerImage and DockerImageTag and ensure that the correct instance of SonarQube is used as the base image.

cc: @redhat-cop/day-in-the-life

Resolves #186
